functions/lib/emails/email_reminder.py

```
@@ -12,127 +12,176 @@
-  body = {
-    "transactionalId": LOOPS_EMAIL_REMINDER_TEMPLATE_ID,
-    "email": email,
-    "dataVariables": {
-        "name": name,
-        "eventName": event_name,
-        "eventId": event_id, 
-        "startDate" : start_date,
-        "endDate": end_date,
-        "location": location
+    try:
+        event_metadata = db.collection(EVENT_METADATA).document(event_id).get()
```

Ruff (0.14.0)

`db` may be undefined, or defined from star imports

(F405)